compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/GraphqlCodeModifier.java

```diff
@@ -46,6 +46,8 @@ public void init(CodeModifierContext modifierContext) {
                                                   SyntaxKind.SERVICE_DECLARATION);
         modifierContext.addSyntaxNodeAnalysisTask(
                 new ModuleLevelVariableDeclarationAnalysisTask(this.modifierContextMap), SyntaxKind.MODULE_VAR_DECL);
+        modifierContext.addSyntaxNodeAnalysisTask(
+                new LocalLevelServiceObjectAnalysisTask(this.modifierContextMap), SyntaxKind.OBJECT_CONSTRUCTOR);
```

Comment:
A better name would be:

```suggestion
                new ObjectConstructorAnalysisTask(this.modifierContextMap), SyntaxKind.OBJECT_CONSTRUCTOR);
```